packages/extension/src/components/pages/AddNewTrustline/AddNewTrustline.tsx

```diff
@@ -258,6 +268,9 @@ export const AddNewTrustline: FC = () => {
   }
 
   if (!isValidIssuer) {
+    chrome.runtime.sendMessage<
+      ReceiveSetTrustlineBackgroundMessage | ReceiveSetTrustlineBackgroundMessageDeprecated
+    >(createMessage({ transactionHash: null, error: new Error('gem_BAD_ISSUER') }));
     return (
       <AsyncTransaction
         title="Incorrect transaction"
```

Comment:
no errors for if (errorValue) { and if (errorDifference) {?

Reply:
- `errorValue` is done on line: 135
- `errorDifference` is done on line: 192
I can move them to the views if you find this easier to read.

Reply:
I think so